File: Exercicio Diagrama de Classes/Classes_Python/pessoa.py
class Pessoa:
    def __init__(self, identificador: str, endereco: str):
        self.identificador = identificador
        self.endereco = endereco

    def get_identificador(self):
        return self.identificador

    def get_endereco(self):
        return self.endereco

    def exibir_dados(self):
        print(f"Identificador: {self.identificador}")
        print(f"Endereço: {self.endereco}")

File: Exercicio Diagrama de Classes/Classes_JS/Pessoa.js
export class Pessoa {
  constructor(identificador, endereco) {
    this.identificador = identificador;
    this.endereco = endereco;
  }

  getIdentificador() {
    return this.identificador;
  }

  getEndereco() {
    return this.endereco;
  }

  exibirDados() {
    return `Identificador: ${this.identificador}\nEndereço: ${this.endereco}`;
  }
}